Update js_gui.md to reflect firmware 1.4 changes

alexeyzakh opened the pull request:
# What's new

- Updated the table with JS view elements to show that they are now all supported.

# Verification 

- Open the changed files and review the changes.

# Checklist (For Reviewer)

- [ ] PR has description of feature/bug or link to Confluence/Jira task
- [ ] Description contains actions to verify feature/bugfix
- [ ] I've built this code, uploaded it to the device and verified feature/bugfix

alexeyzakh committed:
Devboard docs update

Wi-Fi devboard pinout added, updated devboard hardware image, fixed the schematics link, and corrected typos.

hedger committed:
Merge branch 'dev' into developer-docs-editing

alexeyzakh committed:
Update js_gui.md to reflect firmware 1.4 changes

Updated the table with view elements to show that they are now all supported.

alexeyzakh committed:
Merge branch 'developer-docs-editing' of https://github.com/alexeyzakh/flipperzero-firmware into developer-docs-editing

portasynthinca3 commented on documentation/js/js_gui.md:
```diff
@@ -40,24 +40,23 @@ always access the canvas through a viewport.
+* `button_menu`
+* `button_panel`
+* `byte_input`
+* `dialog_ex`
```
JS modules for `dialog_ex` and `file_browser` are named `dialog` and `file_picker` respectively. Please reflect this.

alexeyzakh committed:
Renamed JS GUI views

portasynthinca3 committed:
Merge branch 'dev' into developer-docs-editing

alexeyzakh committed:
Fixed a typo in the JS notification doc

Changed "short" to "long" in the blink() method example to match the description.
